boards/risc-v/mpfs/icicle/configs/opensbi/defconfig

```diff
@@ -53,51 +53,8 @@ CONFIG_MPFS_HART4_SBI=y
 CONFIG_MPFS_OPENSBI=y
 CONFIG_MPFS_UART0=y
 CONFIG_NSH_ARCHINIT=y
-CONFIG_NSH_DISABLE_BASENAME=y
```

Comment:
remove this will use more flash

Reply:
CONFIG_DEFAULT_SMALL disable all tools by default.

Reply:
ok